src/zh/mycomic/src/eu/kanade/tachiyomi/extension/zh/mycomic/MyComic.kt

```diff
@@ -0,0 +1,221 @@
+package eu.kanade.tachiyomi.extension.zh.mycomic
+
+import androidx.preference.ListPreference
+import androidx.preference.PreferenceScreen
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.ConfigurableSource
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import keiyoushi.utils.firstInstance
+import keiyoushi.utils.getPreferencesLazy
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonArray
+import kotlinx.serialization.json.jsonObject
+import kotlinx.serialization.json.jsonPrimitive
+import okhttp3.HttpUrl
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import uy.kohesive.injekt.injectLazy
+
+class MyComic : ParsedHttpSource(), ConfigurableSource {
+    override val baseUrl = "https://mycomic.com"
+    override val lang: String
+        get() = "zh"
+    override val name: String
+        get() = "MyComic"
+    override val supportsLatest: Boolean
+        get() = true
+
+    override fun headersBuilder() = super.headersBuilder().add("referer", baseUrl)
```

Comment:
Should be more similar to how browsers do it:
```suggestion
    override fun headersBuilder() = super.headersBuilder().add("Referer", "$baseUrl/")
```